### ISSUE TITLE
magit-blame-mode propertizes everything with magit-blame-heading face

### ISSUE THREAD

**Issue by alezost** [magit/magit#1761]
(I'm on the "next" branch)

I see 2 issues with `magit-blame-format-heading` function:

1. It uses a non-existing `magit-blame-time` face instead of
  `magit-blame-date` one.

2. The main problem is: the whole blame string is always
   propertized with `magit-blame-heading` face (other faces are
   ignored).  For example, the following code:

```elisp
(magit-blame-format-heading
 '(:hash "eda7de2117946bb7d4633af43a7bc36cec182cdb"
   :author "Alex Kost" :author-time 1500000000.0 :author-tz 100
   :committer "Alex Kost" :committer-time 1500000000.0 :committer-tz 100
   :summary "foo"))
```

evaluates to something like this:

```
#("Alex Kost            2017-07-14 06:40 foo
" 0 9 (face magit-blame-heading) 9 20 (face magit-blame-heading) 20 21 (face magit-blame-heading) 21 37 (face magit-blame-heading) 37 38 (face magit-blame-heading) 38 41 (face magit-blame-heading) 41 42 (face magit-blame-heading))
```

**Comment by tarsius** [comment]
Fixed in 9332dbab34116be862fc38b6a18ea970c8840dfb and d2414ff1370e6380111931383161ebc576559078. Looks like I just assumed `format-spec` would do the right thing.

**Comment by alezost** [comment]
Wow, thanks for fixing it so quickly!
